examples/global_deploy.rs

```diff
@@ -0,0 +1,53 @@
+use near_api::*;
+use near_primitives::hash::hash;
+
+#[tokio::main]
+async fn main() {
+    let network = near_workspaces::sandbox().await.unwrap();
+    let account = network.dev_create_account().await.unwrap();
+    let target_account = network.dev_create_account().await.unwrap();
+    let network = NetworkConfig::from(network);
+
+    let signer = Signer::new(Signer::from_workspace(&account)).unwrap();
+
+    let code: Vec<u8> = include_bytes!("../resources/counter.wasm").to_vec();
+    let contract_hash = hash(&code);
+
+    Contract::deploy_global_contract_code(code.clone())
+        .as_hash(account.id().clone())
```

Comment:
nit: looks like a typo and that it hasn't been run

Reply:
Oh, well, i think i get it. I'd split it into 2 methods call chain:

`as_hash().signer_id()`